doc/nrf/links.txt

```diff
@@ -1078,3 +1078,5 @@
 .. _`Wi-Fi Alliance`: https://www.wi-fi.org/
 
 .. _`IEEE 802.11 Working Group`: https://www.ieee802.org/11/
+
+.. _`BSEC2`: https://github.com/boschsensortec/Bosch-BSEC2-Library
```

Comment:
The links are sorted according to the source. Move this under the heading ### Source: github.com, as the last item.